### DISCUSSION TITLE
Execute normal markdown codeblocks?

### DISCUSSION THREAD

**Discussion by d-chambers** [quarto-dev/quarto-cli#2935] [Q&A] [answered]
Hi,

Is there a way to tell quarto to execute normal md codeblocks?

```
```python
print('hey')
'''
```

Rather than having to surround the environment with curly brackets like this?

```
```{python}
print('hey')
'''
```
(I had to use the bottom backticks as quotation marks to render properly)

I want to do this so that I can use a quarto file as a project readme and have github do the proper code formatting. 

Thanks!

**Comment by cscheid** [comment, marked as answer]
You can use quarto to generate github markdown:

### test.qmd

````
---
format: gfm
---

```{python}
print('Hey')
```
````

will produce test.md:

````
``` python
print('Hey')
```

    Hey
````

which is what I think you're looking for.

**Reply by d-chambers** [reply]
Ya that will work, thanks!
